src/schema/utils.js

```diff
@@ -30,3 +35,16 @@ export function backlinkTable(tableSchema) {
 export function getBacklinkTableName(tableName) {
   return tableName + BACKLINK_TABLE_POSTFIX
 }
+
+export const customJson = customType({
+  dataType() {
+    return 'text'
+  },
+  fromDriver(value) {
+    // @ts-ignore
+    return JSON.parse(value)
+  },
+  toDriver(value) {
+    return JSON.stringify(value)
+  },
+})
```

Comment:
```suggestion
export const customJson =
  /** @type {ReturnType<typeof customType<{ data: import('../generated/rpc.js').Invite_ProjectInfo }>>} */ (
    customType({
      dataType() {
        return 'text'
      },
      fromDriver(value) {
        // @ts-ignore
        return JSON.parse(value)
      },
      toDriver(value) {
        return JSON.stringify(value)
      },
    })
  )
```

Reply:
this is used for other schema things so would have to work with a generic instead

https://github.com/digidem/mapeo-core-next/blob/209/client-add-project/src/schema/schema-to-drizzle.js#L59

https://github.com/digidem/mapeo-core-next/blob/209/client-add-project/src/schema/schema-to-drizzle.js#L81

Reply:
can't figure out how to do this with generics so going to avoid diving into a rabbit hole and keep my ugly runtime checks for now 😄

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/215#discussion_r1309172868